<launch>
  <include file="$(find gazelle_sim_ltu_reu)/launch/ltu_reu.launch" />

  <node name="follow_line" pkg="gazelle_sim_ltu_reu" type="follow_line.py" output="screen">
    <param name="imgtopic_name" type="str" value="robot1/camera/image_raw"/>
  </node>
</launch>
